base/gmp.jl

```diff
@@ -113,57 +115,56 @@ module MPZ
 #   and `add!(x, a) = add!(x, x, a)`.
 using Base.GMP: BigInt, Limb
 
-const mpz_t = Ptr{BigInt}
-const bitcnt_t = Culong
+const Mpz_t = Ref{BigInt}
```

Comment:
I find `Mpz_t` a bit ugly as these are initials. I think this is an unrelated change, so you could do that in another PR, but if you really insist to change it now, I think I would prefer `MPZ_t`. Also I had introduced `bitcnt_t` to make it easier to match the signatures in this `MPZ` module with the GMP docs (as there had been some mistakes previously with using raw types like `Culong`), so I don't understand why you remove it now without giving motivation in an unrelated PR.